vita3k/crypto/src/hash.cpp

```diff
@@ -25,7 +25,7 @@ extern "C" {
 Sha256Hash sha256(const void *data, size_t size) {
     Sha256Hash hash;
     SHA256_CTX sha_ctx = {};
-    sha256_init(&sha_ctx);
+    sha256_init_one(&sha_ctx);
```

Comment:
Why do we need to change the name of the function?

Reply:
there was a double definition error when building

Reply:
I think you should revert the `crypto-algorithms` module.

Reply:
and what would that give us?

Reply:
I see that you swap out the `crypto-algorithms` module and you only rename the function. There is no new functionality.

About double definition error, we have `sha256_init` in `external/vita-toolchain/src/sha256.c` and in `external/crypto-algorithms/sha256.c`.

- Maybe someone extracted that code before.
- Let's see if we need that `crypto-algorithms` anymore. Is it safe to remove it?
- If not, I think we should create PR to `vita-toolchain` to add using features from `crypto-algorithms` and remove `crypto-algorithms` for good.

Reply:
Why build agents are still able to build if that error is a problem?

Reply:
there is sha256_init in libtomcrypt module too, which caused double definition. i changed name so we don't have that anymore. crypto-algoritms is used in renderer.cpp